Assets/Source/Components/Player/PlayerBehavior.cs

```diff
@@ -99,7 +110,16 @@ public void ReactToProjectileHit(Collision2D collision, int baseDamage)
             if (!collision.otherCollider.gameObject.name.Equals(bulletPrefab.name))
             {
                 actorBehavior.Health -= baseDamage;
-                cameraEffector.TriggerImpulse1();
+
+                if (collision.otherCollider.transform.position.x <= transform.position.x)
```

Comment:
It's not a big deal but we shouldn't be in the habit of using any sort of equals comparison with floats since they tend to be imprecise and its just a wasted check.

Reply:
i promis i;ll fix dis in my current branch